webapp/src/modules/address/reducer.js

```diff
@@ -136,6 +156,26 @@ export function addressReducer(state = INITIAL_STATE, action) {
         }
       }
     }
+    case CREATE_ESTATE_SUCCESS: {
+      const { id, owner, data } = action.estate
+      const parcelIds = new Set(
+        data.parcels.map(({ x, y }) => buildCoordinate(x, y))
+      )
+
+      return {
+        ...state,
+        data: {
+          ...state.data,
+          [owner]: {
+            ...state.data[owner],
+            parcel_ids: [
+              ...state.data[owner].parcel_ids.filter(x => !parcelIds.has(x))
```

Comment:
filter already returns a new array so the `[...]` is not necessary

Reply:
True! removed 539d617